libs/utils/utils.js

```diff
@@ -400,6 +400,10 @@ export function decorateSVG(a) {
   try {
     const textContentUrl = new URL(sanitizedTextContent);
     const hrefUrl = new URL(href);
+    
+    // Making sure all svg URLs are relative.
+    img.src = img.src.replace(textContentUrl?.origin, '');
```

Comment:
Why would we not fix img.src when we actually make the image?

Reply:
1. I wanted to reuse `textContentUrl` that is defined at 401 line to make this easy.
2. I wanted to keep localizeLink() code so it is blessed by all the Janaki's logic.
3. There was js error when I was trying it out before `try` statement, I thought it was from `new URL(sanitizedTextContent)` but then I figured it out while I was writing this reply.. so forget about this.

I will update this as.
```
img.src = sanitizedTextContent.replace(new URL(sanitizedTextContent).origin, '');
```